### PR TITLE
prepare v1.0.0 release

### PR DESCRIPTION
**Pull request by aknuds1**

### CODE REVIEW

**Comment by aknuds1** [comment]
@Darksonn How do I compile the changelog?

**Comment by Darksonn** [comment]
Read [the list of commits](https://github.com/tokio-rs/bytes/commits/master) backwards until the last preparation PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.0.0 (December 19, 2020)
```
Comment (carllerche) [inline CHANGELOG.md:1]:
```suggestion
# 1.0.0 (unreleased)
```

File: Cargo.toml [hunk 2]
```diff
@@ -19,7 +19,6 @@ readme = "README.md"
 keywords = ["buffers", "zero-copy", "io"]
 categories = ["network-programming", "data-structures"]
 edition = "2018"
-publish = false
```
Comment (carllerche) [inline Cargo.toml:22]: Revert this line.

**Review by carllerche — CHANGES_REQUESTED** [review]
Hey! Thanks for doing this. We are going to hold off doing the actual release until Tokio is ready for the release as well. However, we can merge most of this :+1: Some thoughts inline.

**Review by carllerche — APPROVED** [review]

**Pull request merged**